docs/user/explanations/aravis-detector.md

```diff
@@ -0,0 +1,102 @@
+:::
+
+The rest of the plugins can then be connected to the aravis-detector. Odin allows plugins to be connected in a tree structure, where plugins can be chained one to another or multiple plugins can be connected into one. For the default aravis plugin config file the live view and hdf plugins are both directly connected to it, similar to plugins 1 and 3 in the following example:
+
+```json
+{   "plugin": {
+        "connect": {
+            "index": "<other plugin 1>",
+            "connection": "aravis"
+}},
+    "plugin": {
+        "connect": {
+            "index": "<other plugin 2>",
+            "connection": "<other plugin 1>"
+}},
+    "plugin": {
+        "connect": {
+            "index": "<other plugin 3>",
+            "connection": "<aravis>"
+}}}
+```
+
+## Plugin Configurations
+
+Different configurations can be passed at start up using the same config file from the setup process. The syntax used is:
+
+```json
+{ "aravis":{
+        "list_devices": true,
+        "compression": "none",
+        "dataset": "data",
+        "ip_address": "169.254.186.47"
+}}
+```
+
+The following list is a complete set of available plugin configuration and their aliases:
+
+| Config | Description| Default value |
+|--------|------------|---------------|
+| ip_address | Connects to the camera with the specified IP address | 127.0.0.1 |
+| exposure_time | Sets the exposure time to the given value | 1000.0 us |
+| frame_rate | Sets the frame rate to the given value | 5 Hz|
+| frame_count | Sets a limit to the number of buffers acquired in continuos mode. 0 for no limit | 0|
+| pixel_format | Sets the pixel format used by the camera | Mono8 |
+| acquisition_mode | Sets the camera's acquisition mode. Not fully implemented | Continuous |
+| aravis_callback | When set to true the camera sends signals when an mage buffers is finished. When set to false the plugin checks for buffers on a regular interval | True |
+| data_set_name | name of the data set all frames are assigned to | No default|
+| file_name | name of the file name all frames are assigned to | No default |
+| compression | compression method used | No default |
+| file_path | file path for all temporary files. Currently used by genicam | No default |
+| start | start camera acquisition of buffers | value is ignored |
+| stop | stop camera acquisition of buffers | value is ignored |
+| list_devices | lists all genicam devices connected | value is ignored |
+| frames | specifies a number of frames to acquire | deprecated, no default |
+
+:::{note}
+The recommended way to change plugin configuration is through the use of the odin-control server.
+:::
+
+## Supported genicam features
+
+As of version 0.0.1, pre-release, the following features are implemented in the Aravis Plugin:
```

Comment:
```suggestion
The following features are implemented in the Aravis Plugin:
```

Prior to first Major version number (1.0) we don't need to specify availability/deprecation of features...